xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/job/question/ExtensionBreakingQuestion.form.vm

```diff
@@ -17,15 +17,16 @@
 ## Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
 ## 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 ## ---------------------------------------------------------------------------
+##!source.syntax=xwiki/2.1
```

Comment:
That's a big change... why exactly do you need that? Can't just use the velocity macros?

Reply:
Where possible I wanted to still use the XWiki macro, which is still the most standard way. It looked like it was easy enough here to change the default syntax of this template and wrap most things in a velocity macro so that I could use the XWiki macro when needed.

I can revert those changes and use the velocimacro if you confirm that it'd be better :+1:

Reply:
I don't have anything against moving to `xwiki/2.1` in general, but you need to keep in mind that the behavior is very different, especially regarding dealing with new lines and stuff like this. It's more probably more dangerous than you think.

Reply:
Reverted the changes. 
In order to use the velocimacro here, I had to upgrade the internal one to feature the two secondary parameters of the XWiki macro.

See changes in https://github.com/xwiki/xwiki-platform/pull/3555/commits/161d3d09292aaf65d9cd0db0882cbf49f897bfbb

Tested on a local instance:
![Screenshot from 2024-12-09 16-06-46](https://github.com/user-attachments/assets/d7b1a9a9-6672-45c0-bbed-6475a92202d7)